### PR TITLE
Add prebuilt exampe

### LINKED ISSUE #55: i don't know how to use an image prebuilt in glance
i don't know how to use an image prebuilt in glance with the packer example

### PR DESCRIPTION
**Pull request by yanndegat**
Fixes #55

### CODE REVIEW

File: examples/public-cluster-cl-prebuilt/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+If you generate a new keypair, put its path in `terraform.tfvars` under variable `public_sshkey` and add it to your ssh-agent:
+```bash
+$ ssh-add terraform_ssh_key
+```
+
+## Build the image
+
+Please refer to the [provided example](../k8s-glance-image/README.md)
```
Comment (thcdrt) [inline examples/public-cluster-cl-prebuilt/README.md]: I don't see in the main.tf when you refer to that image built.

Comment (yanndegat) [reply examples/public-cluster-cl-prebuilt/README.md]: image name no?

File: examples/public-cluster-cl-prebuilt/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Simple k8s cluster from a prebuilt glance image
+
+
+- [Simple k8s cluster](#simple-k8s-cluster)
+    - [Configuration](#configuration)
+    - [Run it](#run-it)
+    
+## Install terraform
+
+You can information on how to install terraform [here](https://www.terraform.io/intro/getting-started/install.html)
```
Comment (strokyl) [inline examples/public-cluster-cl-prebuilt/README.md]: s/You can information/You can read documentation/

File: examples/public-cluster-cl-prebuilt/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+There is an example of var file `terraform.tfvars.sample`.
+
+Copy it under the name `terraform.tfvars` (this allow terraform to autoload those variables)
+
+2. Create a public cloud project on OVH
+
+Follow the [official documentation](https://docs.ovh.com/gb/en/public-cloud/getting_started_with_public_cloud_logging_in_and_creating_a_project/).
+
+You will need to create an Openstack user. You can do so in the "Openstack" part of you cloud project.
```
Comment (strokyl) [inline examples/public-cluster-cl-prebuilt/README.md]: I think it's better to use imperative style everywhere, this lead to shorter documentation:
s/You will need to create an Openstack user. [....]/Create a an Openstack user using the "Openstack" part of youR cloud project/

File: examples/public-cluster-cl-prebuilt/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- module.network
+- module.kube
+[...]
+Terraform has been successfully initialized!
+
+$ terraform apply
+[...]
+helper = Your kubernetes cluster is up.
+
+You can connect in one of the instances:
```
Comment (strokyl) [inline examples/public-cluster-cl-prebuilt/README.md]: s/connect in/connect to/

File: examples/public-cluster-cl-prebuilt/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Run a pod:
+
+    $ ...
+
+Enjoy!
+
+
+```
+
+This should give you an infra with :
```
Comment (strokyl) [inline examples/public-cluster-cl-prebuilt/README.md]: s/with :/with:/

File: examples/public-cluster-cl-prebuilt/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+3. Create or reuse ssh key pair. Carreful this keypair should not be using passphrase !
+
+```bash
+# Generate a new keypair without passphrase
+$ ssh-keygen -f terraform_ssh_key -q -N ""
+```
+
+If you generate a new keypair, put its path in `terraform.tfvars` under variable `public_sshkey` and add it to your ssh-agent:
```
[the remark this answers is not in the document]
Comment (yanndegat) [reply examples/public-cluster-cl-prebuilt/README.md]: relative works

**Pull request merged**